content/en/docs/setup/install/multicluster/single-network/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Single Network Multicluster Deployments
+description: Describes how to deploy a multicluster service mesh on a flat network.
+weight: 10
+list_below: true
+keywords: [multicluster, single-network, primary-cluster, remote-cluster, deployment]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This guide provides all the information needed to complete a {{< gloss >}}multicluster{{< /gloss >}} deployment on a
+[single network.](/docs/ops/deployment/deployment-models/#single-network)
+Go to the [multicluster deployment models page](/docs/ops/deployment/deployment-models/#multiple-clusters)
+for more information.
+
+To start a multicluster deployment, go to the [initial configuration section.](/docs/setup/install/multicluster/single-network/initial-configuration)
+
+After completing the initial configuration, complete the instructions in the
+following sections as many times as needed to add all the
+{{< gloss "cluster" >}}clusters{{< /gloss >}} that you need.
+
+- To add a cluster with a {{< gloss >}}control plane{{< /gloss >}} that engages
+    in cross-cluster load balancing, go to the [add a primary cluster section.](/docs/setup/install/multicluster/single-network/primary)
```

Comment:
Instantly confusing to me: there are more than one primary cluster? Didn't I start this with a single cluster already? Am I load balancing my control plane or my workloads?  Why does it matter?

Reply:
This is a landing page explaining the mechanics of the guide. This higher-level description came about in direct response to TOC feedback saying that the detailed flow being too complex. The deployment models concept covers the answers to your questions.